bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -149,6 +149,34 @@ void ListClusters(bigtable::InstanceAdmin instance_admin, int argc,
 }
 //! [list clusters]
 
+//! [update cluster]
+void UpdateCluster(bigtable::InstanceAdmin instance_admin, int argc,
+                   char* argv[]) {
+  if (argc != 2) {
+    throw Usage{"update-cluster: <project-id> <instance-id> <cluster-id>"};
+  }
+  // CreateCluster or GetCluster first and then modify it
+  bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
+  bigtable::ClusterId cluster_id(ConsumeArg(argc, argv));
+  auto cluster_config =
+      bigtable::ClusterConfig("us-central1-f", 0, bigtable::ClusterConfig::HDD);
+  auto cluster =
+      instance_admin.CreateCluster(cluster_config, instance_id, cluster_id)
+          .get();
+
+  // Modify the cluster
+  cluster.set_default_storage_type(bigtable::ClusterConfig::SSD);
```

Comment:
I think we should show how to change the number of nodes instead, it is a more common use-case for this operation.

Reply:
changed.